tests/Microsoft.Identity.Test.Unit.net45/Microsoft.Identity.Test.Unit.net45.csproj

```diff
@@ -14,6 +14,7 @@
       <Project>{3433eb33-114a-4db7-bc57-14f17f55da3c}</Project>
       <Name>Microsoft.Identity.Client</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Microsoft.Identity.Test.Integration.net45\Microsoft.Identity.Test.Integration.net45.csproj" />
```

Comment:
Let's not do this :). We can share common helpers via the TestCommon assembly.

Reply:
fixed

---
In reply to: [389793604](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1693#discussion_r389793604) [](ancestors = 389793604)